.github/workflows/lint-and-fix.yml

```diff
@@ -0,0 +1,53 @@
+on:
+  pull_request:
+
+jobs:
+  lint-and-fix:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: nodenv/actions-setup-nodenv@v2.0.4
```

Comment:
Any specific reason to try and use v2.0.4, or can we try the latest version that might solve the issue? (If the issue is similar to https://github.com/nodenv/actions/issues/13#issuecomment-749563088 )

Reply:
I _think_ that's what the GitHub Marketplace told me was the latest version. So no, we should use a later version.

Reply:
Fixed by using a different thing.